### PR TITLE
added fill value to Cutout constructor

### PR DESCRIPTION
**Pull request by bfialkoff**
I had the need to use the cutout augmentation, but I wanted to be able to set the color. As it was, the cutout was always black. I thought it would be a useful feature to be able to set the fill_value.

### CODE REVIEW

**Comment by kirillbobyrev** [comment]
Hi! I have recently proposed `Cutout` improvements in #263 and this looks like a subcase for the third improvement (patch modification). I've implemented `CoarseDropout` which deprecates `Cutout` in #264 and I think this patch should at least be applied to `CoarseDropout`. If you could find a way to generalize it and build a high-level interface for patch modifications that could be even better, though I don't have much experience with the project so I don't know what would be the best way to do that.

**Comment by ternaus** [comment]
Does not pass the tests.

**Comment by bfialkoff** [comment]
Yes, I know I made some changes that caused the fail and just haven't
gotten around to correcting it yet.

**Comment by John-Yao** [comment]
hello, I had tried to set the value of Cutout's fill_value . It does work well. All the cut hole  is fill with black.

**Comment by bfialkoff** [comment]
That's strange. Please sharead your code and I'll try and diagnose the
problem.

**Comment by John-Yao** [comment]
import sys
import cv2
import albumentations as A
import matplotlib.pyplot as plt
from mmcv.runner import obj_from_dict
print(A.__version__)
class Augtor(object):
    
    def __init__(self,
                 albumentations = None):
        self.abmtt_transforms = []
        
        if albumentations is not None:
            for _transform in albumentations:                
                self.abmtt_transforms.append(obj_from_dict(_transform, parent=A))
                
    def __call__(self, img):
        img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
        for transform in self.abmtt_transforms: #rgb
            img = transform(image=img)['image']
        img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)
        return img
sampler = Augtor(albumentations=[
               dict(
                    max_h_size=100,
                    max_w_size=100,
                    p=1.0,
                    num_holes = 2,
                    fill_value=[255,0,0],#rgb
                    type='Cutout'
                )
])
I = cv2.imread(img_fname)
sample_img = sampler(I[:1000,:1000])
plt.imshow(cv2.cvtColor(sample_img,cv2.COLOR_BGR2RGB).astype(np.int))
plt.show()

Thanks for your reply! the 'img_fname' have to be set by yourself.
The following image is my result.

image

File: albumentations/core/transforms_interface.py [hunk 1]
```diff
@@ -95,7 +95,11 @@ def targets(self):
     def update_params(self, params, **kwargs):
         if hasattr(self, 'interpolation'):
             params['interpolation'] = self.interpolation
-        params.update({'cols': kwargs['image'].shape[1], 'rows': kwargs['image'].shape[0]})
+        if hasattr(self, 'fill_value'):
+            params['interpolation'] = self.fill_value
```
Comment (albu) [inline albumentations/core/transforms_interface.py]: it's strange to set interpolation as fill value

**Comment by bfialkoff** [comment]
Not strange, just plain incorrect! That's what I get for copy-pasting. I
fixed it and updated the PR.

File: albumentations/core/transforms_interface.py (hunk 1 above)
Comment (bfialkoff) [reply albumentations/core/transforms_interface.py]: Not strange, just plain incorrect! That's what I get for copy-pasting. I
fixed it and updated the PR.

**Pull request merged**